packages/blade/src/components/Checkbox/CheckboxIcon/CheckboxIconWrapperStyles.ts

```diff
@@ -38,10 +38,12 @@ const variants = {
 
 export type CheckboxRectProps = Omit<CheckboxIconProps, 'state'> & {
   isChecked: boolean;
+  isFullRounded?: boolean;
 };
 
 const getCheckboxIconWrapperStyles = ({
   theme,
+  isFullRounded,
```

Comment:
Think you need to update the instances of `isFullRounded`

Reply:
Why?

Reply:
Because we're calling it `isRounded`? [Ref](https://github.com/razorpay/blade/pull/605#discussion_r944391741)